src/harness/testcases/WINNF_FT_S_SSS_testcase.py

```diff
@@ -18,10 +18,67 @@
 
 SAS_CERT = os.path.join('certs', 'sas.cert')
 SAS_KEY = os.path.join('certs', 'sas.key')
+SAS_URL = 'https://fake.sas.url.not.used.org'
+
 
 class SasToSasSecurityTestcase(security_testcase.SecurityTestCase):
   # Tests changing the SAS UUT state must explicitly call the SasReset().
 
+  @winnforum_testcase
+  def test_WINNF_FT_S_SSS_1(self):
+    """New registration with TLS_RSA_WITH_AES_128_GCM_SHA256 cipher.
+
+    Checks that SAS UUT response satisfy cipher security conditions.
+    Checks that a SAS registration with this configuration succeed.
+    """
+    self.doSasTestCipher('AES128-GCM-SHA256',
+                         client_cert=SAS_CERT, client_key=SAS_KEY,
+                         client_url=SAS_TH_URL)
```

Comment:
Should we make the CA certificate chain also as one of the arguments?

Reply:
The CA certificate chain is only related to the server, so I think it should not change across testcases (in particular these ones where we test only cipher).

Reply:
ok, agreed.